### PR TITLE
feat: Cypress tests - pass theme parameter

### PR DESCRIPTION
**Pull request by arjun-at-roam**
## PR Area

- [x] Testing

## Type of change

_What kind of change does this PR introduce?_

- [x] Feature

## Description

* pass in theme to `visitAndInjectWidget` function

## Security & risks

- [ ] Adds new third party libraries
- [ ] Adds new `.env` variables
- [ ] Integrates with Third-party apis (dependency must be resilient)
- [ ] Has specific [OWASP security concerns](https://owasp.org/www-project-top-ten/) - list them: (also see [Fixing OWASP Top 10](https://nodegoat.herokuapp.com/tutorial))

## Pre-publish checklist

- [x] I have assigned myself to the ticket
- [x] I have performed a self-review of my code
- [ ] I have performed relevant browser testing
- [ ] I have performed keyboard testing of new or modified interactive components

## Reviewer checklist

- [ ] I have run the code (if applicable)
- [ ] I have reviewed and run any tests (if applicable)

### CODE REVIEW

File: cypress/page-objects/merchantProductPage.ts [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+    theme = undefined,
+    darkMode = false,
+  }: {
+    url: string
+    targetElement: string
+    position?: string
+    country?: 'nz' | 'au'
+    theme?: 'farmers' | 'qmc' | 'humm90' | undefined
+    darkMode?: boolean
+  }) {
```
Comment (ghost) [inline cypress/page-objects/merchantProductPage.ts]: Nit: consider creating an interface for these parameters to make this more readable

**Review by ghost — APPROVED** [review]
Looks great 🚀

File: cypress/page-objects/merchantProductPage.ts (hunk 1 above)
Comment (arjun-at-roam) [reply cypress/page-objects/merchantProductPage.ts]: Great call!

File: cypress/models/visitAndInjectWidget.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import CountryEnum from 'cypress/models/enum/CountryEnum'
+import ThemeEnum from 'cypress/models/enum/ThemeEnum'
+
+export interface visitAndInjectWidgetProps {
```
Comment (ghost) [inline cypress/models/visitAndInjectWidget.ts]: Could you please PascalCase this interface?

**Review by ghost — APPROVED** [review]

**Pull request merged**